main.go

```diff
@@ -406,6 +411,20 @@ func mustInitHelmCache(maxSize int, itemTTL, purgeInterval string) (*cache.Cache
 	return cache.New(maxSize, interval), ttl
 }
 
+func mustInitOIDCAuthenticator() *oci.OIDCAuthenticator {
+	capacity := oci.DefaultAuthCacheCapacity
+	disabled, found := os.LookupEnv("LOGIN_CACHE_DISABLED")
+	if found && disabled == "true" {
+		capacity = -1
+	}
```

Comment:
Please implement this as a feature flag called `CacheLoginTokens` enabled by default.